### DISCUSSION TITLE
Suggestion: Generic setup guides (with diagrams)

### DISCUSSION THREAD

**Discussion by onyxraven** [ExistentialAudio/BlackHole#140] [General]
This is a great project! But its setup is quite complex. There are a lot of steps, and the step by step is a little tough to follow.

So I'm wondering, could we get some updated guides on some of the common use cases, along with some screenshots and potentially diagrams explaining what's going on?

I'm thinking guides that cover some of the common cases in Issues here, such as: Spotify or Youtube audio (music) In, Microphone In, Monitor Music locally (avoid feedback), send mix to Zoom/Skype

I personally have a *bit* of a hard time following just word examples and screenshots and diagrams (showing how the aggregate/multiout devices, and blackhole work and connect up together) would super help!

**Comment by devinroth** [comment]
I've been working on setup guides. Any feedback is appreciated.

https://existential.audio/blackhole/support/
